ceph/rados/utils.py

```diff
@@ -45,7 +45,6 @@ def set_osd_devices_unmanaged(ceph_cluster, osd_id, unmanaged):
             break
 
     if not service_name:
-        log.error(f"No orch service found for osd: {osd_id}")
```

Comment:
@SrinivasaBharath please explain why we are making this change?

Reply:
It is a mistake from my side. I reverted the changes.